newIDE/app/src/Export/LocalExporters/LocalFileSystem.js

```diff
@@ -31,7 +31,7 @@ export default {
     }
   },
   getTempDir: function() {
-    return os.tmpdir();
+    return path.join(os.tmpdir(), `GDTMP-${os.userInfo().username}`);
```

Comment:
Each call to `userInfo` can [throw a `SystemError` exception](https://nodejs.org/api/os.html#os_os_userinfo_options). You must `catch`  (try/catch) the exception and handle it (by using an empty suffix for example). 

Otherwise, GDevelop will crash on systems that don't have a username.

Reply:
OK

Reply:
Fixed

Reply:
not like you want but I think it's fixed